src/Control/Carrier/FossaApiClient.hs

```diff
@@ -63,4 +63,7 @@ runFossaApiClient apiOpts =
           UploadLicenseScanResult signedUrl licenseSourceUnit -> LicenseScanning.uploadLicenseScanResult signedUrl licenseSourceUnit
           UploadFirstPartyScanResult signedUrl fullSourceUnits -> LicenseScanning.uploadFirstPartyScanResult signedUrl fullSourceUnits
           GetAnalyzedPathRevisions projectRevision -> LicenseScanning.alreadyAnalyzedPathRevision projectRevision
+          -- Reachability
```

Comment:
[Optional]: I think the `-- Reachability` comment is unnecessary as it can be implied by the function name